diff -urN mp3splt-2.6.orig/doc/Makefile.am mp3splt-2.6/doc/Makefile.am
--- mp3splt-2.6.orig/doc/Makefile.am	2014-03-23 13:08:20.175003833 +0100
+++ mp3splt-2.6/doc/Makefile.am	2014-03-23 13:08:40.368337271 +0100
@@ -1,7 +1,7 @@
 EXTRA_DIST = mp3splt.1 manual.html
 man_MANS = mp3splt.1
 
-install-exec-hook:
+install-data-hook:
 if DO_OGG_SYMLINK
 	-cd $(DESTDIR)/$(mandir)/man1 && $(LN_S) mp3splt.1 oggsplt.1
 endif
