diff -Naur mp3splt-2.0/mp3.c mp3splt-2.0-p/mp3.c
--- mp3splt-2.0/mp3.c	Thu Jan  8 15:51:24 2004
+++ mp3splt-2.0-p/mp3.c	Fri Jan 16 00:09:34 2004
@@ -529,9 +529,11 @@
 				{
 					while (state->bytes < begin)
 					{
+						off_t to_read;
+
 						if (feof(state->file_input))
 							return -1;
-						off_t to_read = (begin - state->bytes);
+						to_read = (begin - state->bytes);
 						if (to_read > MAD_BSIZE)
 							to_read = MAD_BSIZE;
 						if ((state->data_len = fread(state->inputBuffer, 1, to_read, state->file_input))<=0)
